src/main/scala/com/atomist/rug/runtime/js/RugContext.scala

```diff
@@ -44,4 +58,6 @@ object LocalRugContext extends RugContext {
     * Id of the team we're working on behalf of
     */
   override def teamId: String = "LOCAL_CONTEXT"
+
+  override def contextRoot(): GraphNode = ???
```

Comment:
Does the not implemented here indicate it is not implemented yet or there is no valid implementation. If the latter, perhaps contextRoot does not belong in the parent class?